compiler/rustc_resolve/src/lib.rs

```diff
@@ -913,6 +913,8 @@ pub struct Resolver<'a> {
     label_res_map: NodeMap<NodeId>,
     /// Resolutions for lifetimes.
     lifetimes_res_map: NodeMap<LifetimeRes>,
+    /// Mapping from generics def-id to RPIT copied generic def-id
```

Comment:
As noted above, expand this comment to explain why we would remap -- probably with a simple example